lib/productions/attribute.js

```diff
@@ -75,7 +75,9 @@ export class Attribute extends Base {
     yield* this.extAttrs.validate(defs);
     yield* this.idlType.validate(defs);
 
-    if (["sequence", "record"].includes(this.idlType.generic)) {
+    if (
+      ["async iterable", "sequence", "record"].includes(this.idlType.generic)
```

Comment:
It indeed doesn't make sense but the spec PR does not fix https://whatpr.org/webidl/1397/402a886...0402e85.html#idl-attributes "The type of the attribute ..." part.

Reply:
The spec PR disallows it outright in any "return to JS position" because it does not define IDL->JS conversion (only JS->IDL).

Reply:
I'd say yes as sequence being attribute is disallowed multiple times in the spec (at the attribute definition and also at the sequence definition).